Pipfile

```diff
@@ -4,19 +4,19 @@ verify_ssl = true
 name = "pypi"
 
 [packages]
-twisted = "*"
 scrapy = "*"
-scrapy-sentry = "*"
-city-scrapers-core = {extras = ["azure"],version = "*"}
+scrapy-sentry = {ref = "v1", git = "https://github.com/City-Bureau/scrapy-sentry.git"}
+city-scrapers-core = {ref = "main", git = "https://github.com/City-Bureau/city-scrapers-core.git", extras = ["azure"]}
 python-dateutil = "*"
 scrapy-wayback-middleware = "*"
-chardet = "*"
 pdfminer-six = "*"
-pypiwin32 = {version = "*",sys_platform = "== 'win32'"}
 
 [dev-packages]
 freezegun = "*"
 pytest = "*"
 "flake8" = "*"
 isort = "*"
-black = "==19.10b0"
+black = "*"
+
+[requires]
+python_version = "3.9"
```

Comment:
@SimmonsRitchie you can configure your IDE to automatically add a new line at the end of file on save, so Github won't show the red error icon above. See more here https://thoughtbot.com/blog/no-newline-at-end-of-file

Reply:
Nice catch and great tip! Thanks, @LienDang 😊